docs/book/customer_and_shopuser.rst

```diff
@@ -0,0 +1,83 @@
+.. index::
+   single: Customer and ShopUser
+
+Customer and ShopUser
+=====================
+
+For handling customers of your system **Sylius** is using a combination of two entities - **Customer** and **ShopUser**.
+The difference between these two entities is simple:
+the **Customer** is a guest in your shop and the **ShopUser** is a user registered in the system - they have an account.
+
+Customer
+--------
+
+The Customer entity was created to collect data about non-registered guests of the system - one that has been buying without having an account
```

Comment:
Besides this whole paragraph sounds pretty sinister, like we're some evil corporation which wants to acquire personal data of off everyone 😈